docs/quickstart_guide.md

```diff
@@ -138,7 +138,8 @@ source /etc/profile && go build && sudo ./vhive
 This script stops the existing cluster if any, cleans up and then starts a fresh single-node cluster.
 
 ```bash
-scripts/cloudlab/start_onenode_vhive_cluster.sh <folder to store logs>
```

Comment:
Why are we removing custom log folder argument?

Reply:
on a second thought, it seems like unnecessary complexity